Tests/LightTests.swift

```diff
@@ -36,6 +36,6 @@ class LightTests: XCTestCase {
 	}
 
 	private func newLight() -> Light {
-		return Light(id: "d3b2f2d97452", power: true, brightness: 0.5, color: Color.white(Color.defaultKelvin), label: "Lamp", connected: true, group: nil, location: nil)
+        return Light(id: "d3b2f2d97452", power: true, brightness: 0.5, color: Color.white(Color.defaultKelvin), label: "Lamp", connected: true, group: nil, location: nil, touchedAt: nil)
```

Comment:
How can I make a light target without depending on a given light that I own? I will need to hardcode a serial in, then attempt to fetch that...